src/Goteo/Application/Config.php

```diff
@@ -183,6 +183,9 @@ static public function load($config_file) {
 			View::addFolder(__DIR__ . '/../../../Resources/templates/responsive');
             View::setTheme('responsive');
 
+            \Goteo\Application\View::setTheme('responsive');
+            // we die here and show a formatted error, most likely reason is a database misconfiguration
+			die(\Goteo\Application\View::render('errors/config', ['msg' => $e->getMessage(), 'info' => $info, 'file' => $config_file, 'code' => 500], false));
```

Comment:
@davidbeig wouldn't this be an issue in production?

Reply:
True, i think this one has slipped when merging with the current state of live